src/app/(proper_react)/(redesign)/(public)/breaches/page.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getBreaches } from "../../../../functions/server/getBreaches";
+import { getL10n } from "../../../../functions/server/l10n";
+import { BreachIndexView } from "./BreachIndexView";
+
+export function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: l10n.getString("breach-all-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("breach-all-meta-social-title"),
+      description: l10n.getString("breach-all-meta-social-description"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("breach-all-meta-social-title"),
+      description: l10n.getString("breach-all-meta-social-description"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
```

Comment:
Pretty sure we had a ticket for this (and probably well out of scope here), but IIUC, this will set the OG meta url to the root DOMAIN, not the current breach page.

Reply:
Looks like it does; I just copied it from the existing page. Fixed in e620454c3 and 6adf003db.